src/validations/member.js

```diff
@@ -0,0 +1,34 @@
+const Joi = require('joi');
+
+const validateUpdate = (req, res, next) => {
+  const memberValidation = Joi.object({
+    firstName: Joi.string().min(3).max(50),
+    lastName: Joi.string().min(3).max(50),
+    dni: Joi.number().min(6).max(8),
```

Comment:
careful here. min and max in numbers refer to minimum value and maximum value, it does not refer to the length.

here you can do this:
```
dni: Joi.number()
  .positive()
  .integer().
  .min(10000000)
  .max(99999999)
```

Reply:
yes!